### PR TITLE
update experimental ubuntu20 container to 1.13.2~rc5

### PR DESCRIPTION
**Pull request by Echsecutor**
Signed-off-by: Sebastian Schmittner <sebastian.schmittner@eecc.de>

### CODE REVIEW

File: build/Dockerfile.ubuntu20 [hunk 1]
```diff
@@ -40,8 +40,7 @@ RUN pip3 install -U \
 
 
 RUN apt-get update -y && apt-get install -y \
-    indy-node="1.13.2~rc4" \
-    indy-plenum="1.13.1~rc3" \
+    indy-node="1.13.2~rc5" \
```
Comment (Echsecutor) [inline build/Dockerfile.ubuntu20:43]:
```suggestion
    indy-node="1.13.2~rc5" \
    indy-plenum="1.13.1~rc3" \
```

**Comment by c2bo** [comment]
There seems to be an issue with the tests?
https://github.com/hyperledger/indy-node-container/actions/runs/4082605570/jobs/7037223485#step:9:32
```
{ [112 bytes data]
  * Connection #0 to host localhost left intact
  jq: error (at ledger_state.json:6): Cannot iterate over null (null)
  jq: error (at ledger_state.json:6): Cannot iterate over null (null)
  jq: error (at ledger_state.json:6): Cannot iterate over null (null)
  jq: error (at ledger_state.json:6): Cannot iterate over null (null)
  jq: error (at ledger_state.json:6): Cannot iterate over null (null)
  jq: error (at ledger_state.json:6): Cannot iterate over null (null)
  jq: error (at ledger_state.json:6): Cannot iterate over null (null)
  jq: error (at ledger_state.json:6): Cannot iterate over null (null)
  Node 1 (synced=, write-consensus=, unreachable-nodes=null): null
  Node 2 (synced=, write-consensus=, unreachable-nodes=null): null
  Node 3 (synced=, write-consensus=, unreachable-nodes=null): null
  Node 4 (synced=, write-consensus=, unreachable-nodes=null): null
```

**Comment by c2bo** [comment]
I tried testing locally and the rc5 version seems to work for me. Any ideas how we can debug what exactly is happening with the docker-compose/nodes inside the github worker?

**Comment by c2bo** [comment]
Adding 
`- LEDGER_SEED=000000000000000000000000Steward1`
to the docker-compose (webserver) fixes the behavior. I will push to this branch with a fix

**Review by c2bo — APPROVED** [review]

**Pull request merged**